charts/spiderpool/values.yaml

```
@@ -61,10 +61,10 @@ feature:
 ## @section clusterDefaultPool parameters
 ##
 clusterDefaultPool:
-  ## @param clusterDefaultPool.installIPv4IPPool install ipv4 spiderpool instance
+  ## @param clusterDefaultPool.installIPv4IPPool install ipv4 spiderpool instance. please must set feature.enableIPv4=true
```

```suggestion
  ## @param clusterDefaultPool.installIPv4IPPool install ipv4 spiderpool instance. It is required to set feature.enableIPv4=true
```
You can search and replace `please must` with `It is required to`